pkg/asset/machines/baremetal/hosts.go

```diff
@@ -136,20 +136,26 @@ func Hosts(config *types.InstallConfig, machines []machineapi.Machine) (*HostSet
 		}
 
 		if !host.IsWorker() && numMasters < numRequiredMasters {
+			// NOTE: When the BMO is running on the bootstrap, we don't want this.
```

Comment:
Can be removed now with all the commented code.

Reply:
Done